plugin-server/src/worker/ingestion/event-pipeline/createEventStep.ts

```diff
@@ -1,15 +1,12 @@
 import { PreIngestionEvent } from '../../../types'
 import { LazyPersonContainer } from '../lazy-person-container'
-import { EventPipelineRunner, StepResult } from './runner'
+import { EventPipelineRunner } from './runner'
 
 export async function createEventStep(
     runner: EventPipelineRunner,
     event: PreIngestionEvent,
     personContainer: LazyPersonContainer
-): Promise<StepResult> {
+): Promise<null> {
```

Comment:
why not `void`?

Reply:
Just replicating what it was doing before I think